storage/storage_tests/storage_tests.cpp

```diff
@@ -845,7 +845,7 @@ UNIT_TEST(StorageTest_GetRootId)
                            "id": "Countries",
                            "v": 151227,
                            "g": []
-                         })"));
+                         })"), new TestMapFilesDownloader());
```

Comment:
OK, no leak, but in any case, please, pass test downloader as unique_ptr, not as raw pointer.